experimenter/experimenter.py

```diff
@@ -340,7 +343,7 @@ def generate_baseline_pipelines(self):
         A wrapper function to generate a AutoSklearn pipeline
         TODO: should be deprecated and removed
         """
-        register_primitives()
+        raise Exception("This function is deprecated")
```

Comment:
I had to delete this line because this function doesn't exist anymore.  Not sure when it disappeared, but it was only needed for the AutoSklearn primitive which we aren't using (due to the DNA repo doing that job now)

Reply:
There is a difference between using Auto-sklearn here and in the DNA repo. In DNA, it is doing metalearning. Here, we use Auto-sklearn to actually make a model for the base datasets.

Reply:
Good point @bjschoenfeld.  I thought we were removing it from the experimenter however - I cant see a good use for it in our experiments.